diff --git a/dist/changesets-get-dependents-graph.cjs.js b/dist/changesets-get-dependents-graph.cjs.js
index 3379ebefac1f304e171afb8d8fe1bc248e4dd8d2..8a3d8ab0292f920fc1258615a18413e7b5386239 100644
--- a/dist/changesets-get-dependents-graph.cjs.js
+++ b/dist/changesets-get-dependents-graph.cjs.js
@@ -19,6 +19,8 @@ const getAllDependencies = (config, ignoreDevDependencies) => {
   for (const type of DEPENDENCY_TYPES) {
     const deps = config[type];
     if (!deps) continue;
+    // https://github.com/changesets/changesets/issues/906
+    if (type === "devDependencies") continue;
 
     for (const name of Object.keys(deps)) {
       const depRange = deps[name];
diff --git a/dist/changesets-get-dependents-graph.esm.js b/dist/changesets-get-dependents-graph.esm.js
index 9e07f669b89b9c111b09c6e816f6e70b9ada49d7..b5f1f6aa704623986fa70df497f8d410562b4a20 100644
--- a/dist/changesets-get-dependents-graph.esm.js
+++ b/dist/changesets-get-dependents-graph.esm.js
@@ -10,6 +10,8 @@ const getAllDependencies = (config, ignoreDevDependencies) => {
   for (const type of DEPENDENCY_TYPES) {
     const deps = config[type];
     if (!deps) continue;
+    // https://github.com/changesets/changesets/issues/906
+    if (type === "devDependencies") continue;
 
     for (const name of Object.keys(deps)) {
       const depRange = deps[name];
